test/StreamTests.hs

```diff
@@ -205,4 +209,23 @@ inlineStringsAreParsed = do
         ]
   expected @==? (items ^.. traversed . si_row . ri_cell_row)
 
+untypedCellsAreParsedAsFloats :: IO ()
+untypedCellsAreParsedAsFloats = do
+  -- values in that file are under `General` cell-type and are not marked
+  -- as numbers explicitly in `t` attribute.
+  items <- runXlsxM "data/floats.xlsx" $ collectItems $ makeIndex 1
+  let defCell v = def
+        { _cellValue = Just v
+        }
+      expected =
+        [ IM.fromList [ (1, def & cellValue ?~ CellDouble 12.0) ]
+        , IM.fromList [ (1, def & cellValue ?~ CellDouble 13.0) ]
+        -- cell below has explicit `Numeric` type, while others are all `General`,
+        -- but sometimes excel does not add a `t="n"` attr even to numeric cells
+        -- but it should be default as number in any cases if `t` is missing
+        , IM.fromList [ (1, def & cellValue ?~ CellDouble 14.0 & cellStyle ?~ 1 ) ]
+        , IM.fromList [ (1, def & cellValue ?~ CellDouble 15.0) ]
+        ]
+  expected @==? (_ri_cell_row . _si_row <$> items)
```

Comment:
What about checking the same for the non-streamed parser? It makes sense to ensure that they do the same thing.

Reply:
I wrote integration test around of that since I do not see an opportunity how
to test `parseCell` in more isolated manner, because `parseCell` are defined
deeply in functions of whole archive parsing.

I hope its fine for now to use it that way? I wrote same test for both parsers (Xeno and Cursored).